docs/cicd/others-script-cd-template.sh

```diff
@@ -0,0 +1,52 @@
+# This is just an example workflow for continous deployment.
+# You should customize it to meet your own requirements.
+# Also you should export the following environment variables for Azure/M365 login:
+# AZURE_ACCOUNT_NAME
+# AZURE_ACCOUNT_PASSWORD
+# AZURE_SUBSCRIPTION_ID
+# AZURE_TENANT_ID
+# M365_ACCOUNT_NAME
+# M365_ACCOUNT_PASSWORD
+
+# If the hosting environment is not provisioned, set this environment variable to false.
+# or if it's provisioned and has not updates, set this environment variable to true.     
+# RUN_PROVISION
+
+
+# Setup environment.
+sudo apt install -y nodejs npm curl git
+sudo npm install -g @microsoft/teamsfx-cli 
+
+# Checkout the code.
+git clone {RepositoryEndpoint}
+cd {FolderName}
+
+# Provision hosting environment.
+if [[ "${RUN_PROVISION}" = "true" ]]
+then
+    teamsfx provision --subscription ${AZURE_SUBSCRIPTION_ID}
```

Comment:
when using a local version, this would be aliased like `npm run teamsfx provision...`

Reply:
I'm going to use something like `npx -p @microsoft/teamsfx-cli@^0.2.2 teamsfx provision`. In this way, developers can specify the version of teamsfx-cli then want to use and it's a local version.

Reply:
that's not better than using a local version. developers should be able to lock the version in their package.json and package-lock.json, instead of being forced to update their CI scripts whenever they update the toolchain

Reply:
> that's not better than using a local version. developers should be able to lock the version in their package.json and package-lock.json, instead of being forced to update their CI scripts whenever they update the toolchain

To clarify, the example scripts are provided to help developers build pipelines in scripts. For example, in the case of Jenkins, they may write scripts using teamsfx-cli, this script file will give them a getting started example. So my point is they are not developing a npm package with package.json inside a project but only writing scripts under a specific cicd pipeline context, there's no package.json there. 
I'm not sure if I made myself clean enough, and how do you think about the point I described above?